src/vs/workbench/browser/parts/titlebar/windowTitle.ts

```diff
@@ -173,6 +181,10 @@ export class WindowTitle extends Disposable {
 		}
 	}
 
+	private titleIncludesFocusedView(): boolean {
```

Comment:
I would rather have a `boolean` property that gets updated when the setting changes and once on startup to reduce cost here.